dev/com.ibm.ws.microprofile.health/src/com/ibm/ws/microprofile/health/internal/AppTrackerImpl.java

```diff
@@ -336,7 +334,12 @@ public void applicationStopped(ApplicationInfo appInfo) {
         // Remove the stopped application from the appState map
         lock.writeLock().lock();
         try {
-            appStateMap.remove(appName);
+            String state = getApplicationMBean(appInfo.getDeploymentName());
```

Comment:
The parameter for the getApplicationMBean, should be "appName", instead of "appInfo.getDeploymentName()

Reply:
Updated.